### PR TITLE
login and signup styling

### PR DESCRIPTION
**Pull request by paul1124**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react';
 
 import AppList from './components/AppList/AppList.jsx';
-import Login from './components/Login';
+import Login from './components/Login/Login';
 import Header from './components/Header';
-import Signup from './components/Signup';
+import Signup from './components/Signup/Signup';
```
Comment (terrancexin) [inline client/src/App.jsx:6]: why duplicate paths?

File: client/src/Components/Signup/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          />
+          {/* <br/> */}
+          <label htmlFor="confirm">Confirm Password: </label>
+          <input 
+            type="password"
+            name="confirm"
+            value={confirm}
+            onChange={e => setConfirm(e.target.value)}
+          />
+          {/* <br/> */}
```
Comment (terrancexin) [inline client/src/Components/Signup/Signup.jsx:48]: delete these comments

File: client/src/App.jsx (hunk 1 above)
Comment (paul1124) [reply client/src/App.jsx:6]: I created a folder for each login and signup jsx files to add scss files. Having all of them inside components folder without categorization seemed redundant.

File: client/src/Components/Signup/Signup.jsx (hunk 2 above)
Comment (paul1124) [reply client/src/Components/Signup/Signup.jsx:48]: Didn't notice. Will do!

**Pull request closed without merging**